src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarBattleSessionUtil.java

```diff
@@ -59,14 +59,7 @@ public static void evaluateBattleSessions() {
 							//If any battle points were gained, calculate a result
 							if(siege.getAttackerBattlePoints() > 0 || siege.getDefenderBattlePoints() > 0) {
 								//Calculate result
-								int battlePointsOfWinner;
-								if (siege.getAttackerBattlePoints() > siege.getDefenderBattlePoints()) {
-									battlePointsOfWinner = siege.getAttackerBattlePoints();
-								} else if (siege.getAttackerBattlePoints() < siege.getDefenderBattlePoints()) {
-									battlePointsOfWinner = -siege.getDefenderBattlePoints();
-								} else {
-									battlePointsOfWinner = 0;
-								}
+								int battlePointsOfWinner = calcPointsForBattleSession(siege);
```

Comment:
:construction: 
- This variable stores the siege balance adjustment
- Thus I suggest it be renamed to "int siegeBalanceAdjustment"